airflow/cli/cli_parser.py

```diff
@@ -396,6 +396,7 @@ def positive_int(value):
     ("--ship-dag",), help="Pickles (serializes) the DAG and ships it to the worker", action="store_true"
 )
 ARG_PICKLE = Arg(("-p", "--pickle"), help="Serialized pickle object of the entire dag (used internally)")
+ARG_EXCEPTION_FILE = Arg(("--error-file",), help="File to store task failure error")
```

Comment:
Error vs exception :)

Reply:
i initially stated with exception, then I noticed it is named as error as in `taskinstance.handle_failure` and it's type could be a str or exception :( so i used error everywhere to be consistent with the type. I think error would be more accurate where, but let me know if you still prefer exception.

Reply:
I just mean the variable name is EXCEPTION, but the flag is "error"

Reply:
@houqp Can you address this please. Just a variable name change or description change :)

Reply:
oh my bad, not sure why github is not notifying this comment to me earlier :(